app/login/page.tsx

```diff
@@ -1,12 +1,9 @@
 import LoginPage from "./LoginPage";
-import { AuthProvider } from "../_components/useContext/AuthContext";
 
 export default function Home(): JSX.Element {
   return (
     <div className="absolute overflow-hidden inset-0 bg-gray-200">
-      <AuthProvider>
-        <LoginPage />
-      </AuthProvider>
+      <LoginPage />
```

Comment:
By removing the LoginPage out of the AuthProvider, there is no context for the LoginPage to know if a user is logged in.